src/__experimental__/components/date/date.spec.js

```diff
@@ -308,18 +349,46 @@ describe('Date', () => {
     describe('when the "closeDatePicker" method has been called', () => {
       it('then the visible value should not change', () => {
         const mockDate = getFormattedDate(moment('2012-02-01'));
-        wrapper = mount(<Date value={ mockDate } />);
+        wrapper = mount(<DateInput value={ mockDate } />);
         wrapper.instance().closeDatePicker();
         const input = wrapper.find('input');
 
         expect(input.instance().value).toBe(mockDate);
       });
     });
   });
+
+  describe('when additional validations are provided with the "validations" prop', () => {
+    describe('as a function', () => {
+      it('then these validations should be passed with internal validations to the Textbox Component', () => {
+        wrapper = render({ validations: isNotFirstApr });
+        const { internalValidations } = wrapper.instance().props;
+        expect(wrapper.find(Textbox).props().validations).toEqual([isNotFirstApr, ...internalValidations]);
+      });
+    });
+
+    describe('as an array', () => {
+      it('then these validations should be passed with internal validations to the Textbox Component', () => {
+        wrapper = render({ validations: [isNotFirstApr] });
+        const { internalValidations } = wrapper.instance().props;
+        expect(wrapper.find(Textbox).props().validations).toEqual([isNotFirstApr, ...internalValidations]);
+      });
+    });
+  });
 });
 
+function isNotFirstApr(value) {
```

Comment:
is it possible to mock this function inside the test?

Reply:
you're right, function removed